entrypoint.sh

```diff
@@ -141,18 +141,26 @@ if [[ "${PHP}" == "" ]];then
     exit 1
 fi
 
+RECONFIGURE_PHP_DEFAULT="yes"
```

Comment:
added as security guard so that the `roave-backward-compatibility-check` is executed with the default PHP version which was set via `Dockerfile` as that is the version which ran `composer install` and thus is compatible with all the external tools.

Reply:
See comment above: IMO, we should have a dedicated `roave-backward-compatibility-check` shell script in `$PATH`, where this sort of defaulting happens.

This removes the `@default` PHP requirement too.